src/plugins/action_console_log/src/dummy2.ts

```diff
@@ -0,0 +1,14 @@
+module.exports['dummy2'] = function (parameters: any) {
```

Comment:
I don't see what is the different between this one and the `src/plugins/action_console_dummy_plugin_2/src/dummy2.ts` file. Is this to show a difference on how plugins can export similar function names?

Reply:
Yup, these dummy files shouldn't be in this folder -- will delete. 😊